pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -319,422 +320,469 @@ async def run(self):
         remote_url = remote_url if remote_url[-1] == "/" else f"{remote_url}/"
         optimize_sync = self.optimize
 
+        task = asyncio.ensure_future(self._fetch_remote_urls(remote_url))
```

Comment:
any reason we are using ensure_future here?
A task can run separately but you are awaiting on it here, so it does not bring any benefit.
I think just awaiting on _fetch_remote_urls would suffice.

Reply:
This kind of obsoletes my previous comment, but i'm still happy to have said that. We have broken 3.6 support on that very function before...